### DISCUSSION TITLE
Treeview row height and parent icon

### DISCUSSION THREAD

**Discussion by kgall89** [israel-dryer/ttkbootstrap#384] [Q&A]
Hello all,

I have just started to use ttkbootstrap and my projects typically use a lot of treeview widgets. I am wondering if there is a way to increase the row height for a treeview widget?

This may be solved by the first question but I am also looking to increase the size of the parent open icon, right now it is very small, and im not getting the box around the icon either. This may be a coloring issue but I haven't been able to figure it out quite yet. The ttkcreator does not seem to show the treeview when creating, just a tableview.

<img width="106" alt="Screenshot 2022-12-18 at 10 31 00 PM" src="https://user-images.githubusercontent.com/120901240/208303978-df610530-ae3d-4587-8b8b-128dbd38e71e.png">

Any ideas would be appreciated!

**Comment by israel-dryer** [comment]
Yes, you can increase the row height, but you'll need to create a style for it.  You can do that by subclassing the existing style and adjusting the `rowheight` parameter such as in this contrived example.

```python
import ttkbootstrap as ttk
from ttkbootstrap.constants import *
from random import choice, randint


window = ttk.Window()
window.style.configure('my.Treeview', rowheight=40)

tv = ttk.Treeview(style='my.Treeview', columns=(0, 1, 2))

# add some generic columns
for i, lbl in enumerate(['Name', 'Age', 'Occupation']):
    tv.heading(i, text=lbl)

# add some groups
for i in range(4):
    tv.insert('', END, i, text=f'Group {i}')


# add some random records for each group
for i in range(4):
    for j in range(3):
        name = choice(['Phillipe', 'Max', 'Antony', 'Tina', 'Alexa'])
        age = randint(25, 60)
        occupation = choice(['Developer', 'Accountant', 'Nurse', 'Secretary'])
        tv.insert(i, END, values=(name, age, occupation))

tv.pack(fill=X, expand=YES)
window.mainloop()
```

<img width="404" alt="image" src="https://user-images.githubusercontent.com/42584742/208308552-ce977e9b-ce96-4681-8b36-99a81682c87d.png">

The only way to change the size of the indicator is to create and map new assets to the various indicator states within the style.

**Reply by kgall89** [reply]
Thank you for your response! And thank you for creating this!
